prompt/prompt-executor/prompt-executor-clients/prompt-executor-ollama-client/src/commonMain/kotlin/ai/koog/prompt/executor/ollama/client/OllamaModelCard.kt

```diff
@@ -0,0 +1,26 @@
+package ai.koog.prompt.executor.ollama.client
+
+import ai.koog.prompt.llm.LLMCapability
+import ai.koog.prompt.llm.LLMProvider
+import ai.koog.prompt.llm.LLModel
+
+@ConsistentCopyVisibility
+public data class OllamaModelCard internal constructor(
```

Comment:
ConsistentCopyVisibility is going to be deprecated in future, according to https://kotlinlang.org/api/core/kotlin-stdlib/kotlin/-consistent-copy-visibility/

Are there any reasons why we would want make that ctor internal?